site/gatsby-site/playwright/e2e/landingPage.spec.ts

```diff
@@ -0,0 +1,123 @@
+import { expect } from '@playwright/test';
+import { conditionalIntercept, waitForRequest, test } from '../utils';
+import { format } from 'date-fns';
+
+const now = new Date();
+
+test.describe('The Landing page', () => {
+  test('successfully loads', async ({ page }) => {
+    await page.goto('/');
+  });
+
+  test('Sends a search to the Discover app', async ({ page }) => {
+    await page.goto('/');
+    await page.locator('form#quickSearch input').fill('Test');
+
+    await Promise.all([
+      page.waitForURL(/\/apps\/discover/), // Wait for the URL to match the pattern
+      page.locator('form#quickSearch button[type="submit"]').click() // Trigger the form submission
+    ]);
+
+    await expect(page).toHaveURL(/\/apps\/discover/);
+    await expect(page).toHaveURL(/s=Test/);
+  });
+
+
+  test('Loads the sponsor modals', async ({ page }) => {
+    await page.goto('/');
+    await page.locator('[data-cy="Waking Up Foundation-image"]').scrollIntoViewIfNeeded();
+    await page.locator('[data-cy="Waking Up Foundation-image"]').click();
+    await expect(page.locator('[data-cy="sponsor-modal"]')).toBeVisible();
+  });
+
+  test('Should submit a report through the Quick Add form', async ({ page }) => {
+    await conditionalIntercept(
+      page,
+      '**/graphql',
+      (req) => req.postDataJSON().operationName == 'InsertQuickAdd',
+      { data: { insertOneQuickadd: { __typename: 'Quickadd', _id: '6271a62068d0c59a372b0c09' } } },
+      'InsertQuickAdd'
+    );
+
+    await page.goto('/');
+    await page.locator('[data-cy="quick-add"] [name="url"]').fill('https://example.com');
+    await page.locator('[data-cy="quick-add"] [type="submit"]').click();
+
+    const insertQuickAddRequest = await waitForRequest('InsertQuickAdd');
+    const variables = insertQuickAddRequest.postDataJSON().variables;
+    expect(variables.data.url).toBe('https://example.com/');
+    expect(variables.data.date_submitted).toBe(format(now, 'yyyy-MM-dd'));
+
+    await expect(page.locator('.tw-toast')).toContainText('Report successfully added to review queue. You can see your submission here.');
+  });
+
+
+  test('Should redirect to the account page when logged in',
```

Comment:
@clari182 Please check why this test is skipped on the report.
These are the skipped tests:
<img width="1001" alt="image" src="https://github.com/responsible-ai-collaborative/aiid/assets/6564809/1603a131-b2fd-4f0e-83b4-30fa5ed879e7">